test/relations.test.js

```diff
@@ -2152,11 +2250,21 @@ describe('relations', function() {
       Picture.find({include: 'imageable'}, function(err, pics) {
         should.not.exist(err);
```

Comment:
While you are changing this test, can you please rewrite this line to `if (err) return done(err)` too?